Makefile

```diff
@@ -114,12 +114,7 @@ fmtcheck:
 
 .PHONY: test-windows
 test-windows:
-	go test -short $(race_detector) ./plugins/inputs/ping/...
-	go test -short $(race_detector) ./plugins/inputs/win_perf_counters/...
-	go test -short $(race_detector) ./plugins/inputs/win_services/...
-	go test -short $(race_detector) ./plugins/inputs/procstat/...
-	go test -short $(race_detector) ./plugins/inputs/ntpq/...
-	go test -short $(race_detector) ./plugins/processors/port_name/...
+	go test -short ./...
```

Comment:
Any reason we dropped the race detector?

Reply:
Build with race detector temporary enabled: https://ci.appveyor.com/project/influx/telegraf/builds/36450543
```
go test -short -race ./...
# runtime/cgo
exec: "gcc": executable file not found in %PATH%
?   	github.com/influxdata/telegraf	[no test files]
FAIL	github.com/influxdata/telegraf/agent [build failed]
FAIL	github.com/influxdata/telegraf/config [build failed]
```

Probably related to https://github.com/golang/go/issues/27089

Few days ago I tried with `mingw` installed https://ci.appveyor.com/project/influx/telegraf/builds/36325329
```
make test-windows
go test -short -race ./...
go build runtime/cgo: C:\go\pkg\tool\windows_amd64\cgo.exe: exit status 2
?   	github.com/influxdata/telegraf	[no test files]
FAIL	github.com/influxdata/telegraf/agent [build failed]
FAIL	github.com/influxdata/telegraf/config [build failed]
```

But it works fine on my local Windows. 

@ssoroka if you have any idea how to configure CI for Windows to run tests with `-race` let me know.